hive-llap/llap.sh

```diff
@@ -66,18 +68,17 @@ function configure_yarn_site(){
 function download_init_actions() {
     # Download initialization actions locally. Check if metadata is supplied
     echo "downalod init actions supplied as metadata..."
-    mkdir "${INIT_ACTIONS_DIR}"/hive-llap
-    gsutil -m rsync -r "${INIT_ACTIONS_REPO}/hive-llap/" "${INIT_ACTIONS_DIR}/hive-llap/"
-    find "${INIT_ACTIONS_DIR}" -name '*.sh' -exec chmod +x {} \;
+    mkdir -p "${INIT_ACTIONS_DIR}"
+    gsutil cp "${INIT_ACTIONS_REPO}/hive-llap/start_llap.sh" "${INIT_ACTIONS_DIR}"
+    find "${INIT_ACTIONS_DIR}/start_llap.sh" -name '*.sh' -exec chmod +x {} \;
```

Comment:
Can be replaced with chmod directly since it's a single file now.

Reply:
changed